app/utility/base_obfuscator.py

```diff
@@ -6,7 +8,11 @@ class BaseObfuscator(BaseWorld):
     def run(self, link, **kwargs):
         agent = self.__getattribute__('agent')
         supported_platforms = self.__getattribute__('supported_platforms')
-        if agent.platform in supported_platforms and link.ability.executor in agent.executors:
-            o = self.__getattribute__(link.ability.executor)
-            return o(link, **kwargs)
+        try:
+            if agent.platform in supported_platforms and link.ability.executor in agent.executors:
+                o = self.__getattribute__(link.ability.executor)
+                return o(link, **kwargs)
+        except Exception:
```

Comment:
what type of exception did you find being thrown here?

Reply:
AttributeError: 'Obfuscation' object has no attribute 'cmd' was the specific one being thrown.

Reply:
ah, now that makes sense. the primary executor of your agent must be "cmd" (command prompt). in your except block, do you need to send back the command?

Reply:
Weird -- my comment didn't follow this one. But in essence, the command was already returned at the base of this function anyway. I didn't alter that behavior.

Reply:
nice - thanks!